.github/workflows/docs.yml

```diff
@@ -35,19 +35,42 @@ jobs:
       - uses: dtolnay/rust-toolchain@7b1c307e0dcbda6122208f10795a713336a9b35a
         with:
           toolchain: stable
+      - uses: Swatinem/rust-cache@23bce251a8cd2ffc3c1075eaa2367cf899916d84
```

Comment:
I'm just not sure why don't we have cache in this particular workflow. Although this is unrelated and I can move it to a separate PR.